downloads/id/github-git-cheat-sheet.md

```diff
@@ -1,42 +1,45 @@
 ---
 layout: cheat-sheet
 title: Buku Saku Git GitHub
-byline: Git adalah sistem manajemen revisi terdistribusi bersifat terbuka yang menyediakan aktivitas GitHub di laptop atau komputer pribadi Anda. Buku saku ini meringkas baris perintah instruksi-instruksi Git yang paling sering dipakai sebagai referensi singkat.
+byline: Git adalah sistem manajemen revisi terdistribusi bersifat terbuka yang memfasilitasi aktivitas GitHub di laptop atau komputer pribadi Anda. Buku saku ini meringkas baris perintah instruksi-instruksi Git yang biasa digunakan sebagai referensi singkat.
 leadingpath: ../
 ---
 
 {% capture colOne %}
-## Pasang Git
-GitHub menyediakan klien bagi komputer, termasuk di dalamnya ada antarmuka grafis untuk tindakan repositori yang paling umum dilakukan dan tampilan konsol tempat menuliskan baris perintah untuk melaksanakan tindakan lebih lanjut.
+## Memasang Git
+GitHub menyediakan klien untuk komputer yang di dalamnya termasuk sebuah
+antarmuka grafis untuk tindakan-tindakan yang paling umum dilakukan pada suatu
+repositori dan sebuah edisi konsol pembaruan otomatis dari Git untuk skenario
+lanjutan.
 
 ### GitHub untuk Windows
-http://windows.github.com
+[windows.github.com](http://windows.github.com)
 
 ### GitHub untuk Mac
-http://mac.github.com
+[mac.github.com](http://mac.github.com)
 
 Distribusi Git untuk Linux dan sistem POSIX tersedia di situs resmi Git SCM.
 
-### Git untuk semua media
-https://git-scm.com
+### Git untuk Semua Sistem
+[git-scm.com](https://git-scm.com)
 
-## Konfigurasi alat
+## Konfigurasi Alat
 Konfigurasi informasi pengguna untuk semua repositori lokal
 
 ```$ git config --global user.name "[nama]"```
 
-Menata nama yang ditautkan dalam pencatatan _commit_ Anda
+Mengatur nama yang ingin ditautkan pada transaksi _commit_ Anda
 
 ```$ git config --global user.email "[alamat surel]"```
 
-Menata alamat pos-el yang ditautkan dalam pencatatan _commit_ Anda
+Mengatur alamat surel yang ingin ditautkan pada transaksi _commit_ Anda
 
-## Buat repositori
-Memulai repositori baru atau mengambilnya dari tautan yang sudah ada
+## Membuat Repositori
+Mulai repositori baru atau dapatkan satu dari URL yang sudah ada
```

Comment:
I think we do not need "satu"  because it is obvious that One URL is One Repository